src/csharp/Grpc.Core.Api/ServerServiceDefinition.cs

```diff
@@ -18,33 +18,31 @@
 
 using System;
 using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Linq;
-using Grpc.Core.Interceptors;
-using Grpc.Core.Internal;
-using Grpc.Core.Utils;
 
 namespace Grpc.Core
 {
     /// <summary>
-    /// Mapping of method names to server call handlers.
+    /// Stores mapping of methods to server call handlers.
     /// Normally, the <c>ServerServiceDefinition</c> objects will be created by the <c>BindService</c> factory method 
     /// that is part of the autogenerated code for a protocol buffers service definition.
     /// </summary>
     public class ServerServiceDefinition
     {
-        readonly ReadOnlyDictionary<string, IServerCallHandler> callHandlers;
+        readonly IReadOnlyList<Action<ServiceBinderBase>> addMethodActions;
 
-        internal ServerServiceDefinition(Dictionary<string, IServerCallHandler> callHandlers)
+        internal ServerServiceDefinition(List<Action<ServiceBinderBase>> addMethodActions)
```

Comment:
Why not just use an IEnumerable here? I don't think you're doing anything that requires a List since you're just iterating through the elements.

Reply:
It's internal so it doesn't matter. It can change in the future if passing in an enumerable makes more sense.